common/src/main/java/com/microsoft/identity/common/internal/cache/SchemaUtil.java

```diff
@@ -154,10 +154,26 @@ public static String getIdentityProvider(final String idTokenString) {
                 final Map<String, String> idTokenClaims = idToken.getTokenClaims();
 
                 if (null != idTokenClaims) {
-                    final String aadVersion = idTokenClaims.get(AuthenticationConstants.OAuth2.AAD_VERSION);
-                    if (!TextUtils.isEmpty(aadVersion) && aadVersion.equalsIgnoreCase("1.0")) {
+                    final String aadVersion = idTokenClaims.get(
+                            AuthenticationConstants.OAuth2.AAD_VERSION
+                    );
+                    if (!TextUtils.isEmpty(aadVersion) &&
+                            aadVersion.equalsIgnoreCase("1.0")) {
+
                         idp = idTokenClaims.get(AzureActiveDirectoryIdToken.IDENTITY_PROVIDER);
-                    } else if (!TextUtils.isEmpty(aadVersion) && aadVersion.equalsIgnoreCase("2.0")) {
+
+                        // For home accounts idp claim is not available, use iss claim instead.
+                        if(TextUtils.isEmpty(idp)){
+                            Logger.info(
+                                    TAG + ":" + methodName,
+                                    "idp claim was null, using iss claim")
+                            ;
```

Comment:
Just an odd formatting artifact - this semicolon looks lonely 🤗

Reply:
Haha typo 😆